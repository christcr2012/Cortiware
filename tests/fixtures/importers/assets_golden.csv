id,type,size,idTag
FIXTURE-001,rolloff,30yd,TAG-A1
FIXTURE-002,port-a-john,std,TAG-B2

