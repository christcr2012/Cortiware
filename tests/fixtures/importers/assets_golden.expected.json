[
  {
    "id": "FIXTURE-001",
    "type": "rolloff",
    "size": "30yd",
    "idTag": "TAG-A1",
    "orgId": "test-org"
  },
  {
    "id": "FIXTURE-002",
    "type": "port-a-john",
    "size": "std",
    "idTag": "TAG-B2",
    "orgId": "test-org"
  }
]

